superset/assets/javascripts/explorev2/components/ExploreViewContainer.jsx

```diff
@@ -74,6 +75,25 @@ class ExploreViewContainer extends React.Component {
   }
 
   render() {
+    const errors = [];
+    for (const fieldName in this.props.fields) {
+      const field = this.props.fields[fieldName];
+      if (field.validationErrors && field.validationErrors.length > 0) {
+        errors.push(
+          <div key={fieldName}>
+            <strong>{`[ ${field.label} ] `}</strong>
+            {field.validationErrors.join('. ')}
+          </div>
+        );
+      }
+    }
+    let errorMessage;
+    if (errors.length > 0) {
+      errorMessage = (
+        <div style={{ textAlign: 'left' }}>{errors}</div>
```

Comment:
curious why we need this `textAlign: left` here... is it because tooltips are styled to align in the center?